ui/src/app/components/add-new-column/add-new-column.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { ColLength, Dialect } from 'src/app/app.constants';
+import { IColumn, ISpannerDetails } from 'src/app/model/conv';
+import { IAddColumn } from 'src/app/model/update-table';
+import { DataService } from 'src/app/services/data/data.service';
+import { TargetDetailsFormComponent } from '../target-details-form/target-details-form.component';
+interface IAddColumnProps {
+  dialect: string
+  tableId: string
+}
```

Comment:
Do we follow a convention on where to declare interfaces?

Reply:
Moved to models